packages/app-project/pages/_app.js

```diff
@@ -69,11 +70,12 @@ export default class MyApp extends App {
 
   render () {
     const { Component, pageProps, theme } = this.props
+    const mergedThemes = merge({}, base, theme)
```

Comment:
We're careful on this one, so I don't worry about it unduly, but we should probably freeze this at some point